### ISSUE TITLE
Clean Up Repo

### ISSUE THREAD

**Issue by michaelamiller12** [aodn/rimrep-examples#13]
Given this repo is now public and provided as training documentation to data providers and clients, we should clean it up and make sure the only information within are notebook examples applicable to the DMS.

- [x] Are all folders still necessary?
- [x] remove renv
- [x] the title "scripts_poc-data-api" should be updated to "scripts" or "notebook examples" - these examples have access via S3, not api at the moment
- [x] Re-organise to something that is easily understandable by people so they know exactly where to go for what. Having a project for all examples may not make the most sense moving forward?
- [x] michaela to add pygeoapi access examples into repo
- [x] In the `extracting_spatial_data_GBR_Features.rmd` there is a line saying "We can extract data from the AIMS Sea Surface Water Temperature dataset" but this does not apply to this example, only data is the GBR features from what i can see
- [x] Surely the `useful_spatial_functions script` can be put in the notebook for using gbr features to grab aims data (`extracting_water_temperature_gbr_features.rmd`) since that is what its used for? I'm unclear as to why it is separate?

**Comment by michaelamiller12** [comment]
I can't add @lidefi87 as an assignee - does Denisse have access to our github?

**Comment by lidefi87** [comment]
I just added myself. All folders are needed with the exception of the `renv` folders, which will be removed because it has caused some many issues. I will be working on this now and push changes when I am done

**Comment by lidefi87** [comment]
I just created a pull request #14 addressing some of the items in this list, but I am including some additional comments below:
> * [ ]  Are all folders still necessary?
The `images` folder contain an image used in one of the `R` examples. 
All folders within the now named `scripts` folder are needed because they contain the necessary information to render the "GitHub friendly" markdowns properly (files ending in .md).
I am not sure about the `RLS` folder, the last commit for this folder came from Nick some time last year. 
> * [ ]  remove renv
This has been done in my new pull request and replaced by the `checking_libraries` function. 
> * [ ]  the title "scripts_poc-data-api" should be updated to "scripts" or "notebook examples" - these examples have access via S3, not api at the moment
This has been done in my new pull request.
> * [ ]  Re-organise to something that is easily understandable by people so they know exactly where to go for what. Having a project for all examples may not make the most sense moving forward?
Everything is documented in the `README` and it makes sense to me, but happy to change it to whatever option is considered more accessible to most people.
> * [ ]  In the `extracting_spatial_data_GBR_Features.rmd` there is a line saying "We can extract data from the AIMS Sea Surface Water Temperature dataset" but this does not apply to this example, only data is the GBR features from what i can see
This has been corrected in the new pull request
> * [ ]  Surely the `useful_spatial_functions script` can be put in the notebook for using gbr features to grab aims data (`extracting_water_temperature_gbr_features.rmd`) since that is what its used for? I'm unclear as to why it is separate?
This is a standalone script that users can call at any time. Its use is shown in just one script, but users can be call it in any notebook or any scripts they develop on their own.

The `Python` notebooks are still pending.

**Comment by michaelamiller12** [comment]
> I just created a pull request #14 addressing some of the items in this list, but I am including some additional comments below:
> 
> > * Are all folders still necessary?
> 
> The `images` folder contain an image used in one of the `R` examples. All folders within the now named `scripts` folder are needed because they contain the necessary information to render the "GitHub friendly" markdowns properly (files ending in .md). I am not sure about the `RLS` folder, the last commit for this folder came from Nick some time last year.
> 
> > * remove renv
> 
> This has been done in my new pull request and replaced by the `checking_libraries` function.
> 
> > * the title "scripts_poc-data-api" should be updated to "scripts" or "notebook examples" - these examples have access via S3, not api at the moment
> 
> This has been done in my new pull request.
> 
> > * Re-organise to something that is easily understandable by people so they know exactly where to go for what. Having a project for all examples may not make the most sense moving forward?
> 
> Everything is documented in the `README` and it makes sense to me, but happy to change it to whatever option is considered more accessible to most people.
> 
> > * In the `extracting_spatial_data_GBR_Features.rmd` there is a line saying "We can extract data from the AIMS Sea Surface Water Temperature dataset" but this does not apply to this example, only data is the GBR features from what i can see
> 
> This has been corrected in the new pull request
> 
> > * Surely the `useful_spatial_functions script` can be put in the notebook for using gbr features to grab aims data (`extracting_water_temperature_gbr_features.rmd`) since that is what its used for? I'm unclear as to why it is separate?
> 
> This is a standalone script that users can call at any time. Its use is shown in just one script, but users can be call it in any notebook or any scripts they develop on their own.
> 
> The `Python` notebooks are still pending.

Amazing!! thank you :) let me know when it's ready to review and I can take a look if you'd like?

**Comment by juliayun23** [comment]
The `RLS` folder seems to be out of the place now. If not needed here, probably can move to https://github.com/aodn/rimrep-examples-private repo. Need to be confirmed by @diodon

**Comment by barbuz** [comment]
The RLS folder contains an example use case built before we started developing our system, and as such it relies on access to external data. It's not necessary, but it could be nice to replicate this with our data once we have ingested everything we need. We already have the geometries, but this is using Real Life Survey fish abundance data from AODN: do you know if this is something that we are planning to ingest @michaelamiller12?

In any case I agree that we can move it to `rimrep-examples-private` for now.

**Comment by michaelamiller12** [comment]
> The RLS folder contains an example use case built before we started developing our system, and as such it relies on access to external data. It's not necessary, but it could be nice to replicate this with our data once we have ingested everything we need. We already have the geometries, but this is using Real Life Survey fish abundance data from AODN: do you know if this is something that we are planning to ingest @michaelamiller12?
> 
> In any case I agree that we can move it to `rimrep-examples-private` for now.

I don't have anything in the system regarding reef life surveys from AODN, I don't think this is a dataset that's been identified by the audit at least. I think its best to move to rimrep-examples-private for now. i can do this today

**Comment by michaelamiller12** [comment]
all tasks have been completed
